src/stactools/palsar2_scansar/constants.py

```diff
@@ -107,15 +120,15 @@
             "title": "linci",
             "type": pystac.MediaType.COG,
             "description": "Local incidence angle (degrees).",
-            "role": ["data", "local-incidence-angle"],
+            "role": ["local-incidence-angle"],
         }
     ),
     "MSK": AssetDefinition(
         {
             "title": "mask",
             "type": pystac.MediaType.COG,
             "description": "Quality Mask",
-            "role": ["data", "data-mask"],
```

Comment:
From our discussion - can add in multiple roles for the asset

Reply:
now added "metadata" to the linci and mask.